cluster-up/cluster/kind-k8s-sriov-1.17.0/certcreator/go.mod

```diff
@@ -1,11 +1,8 @@
-module github.com/kubevirt/csrcreate
+module kubevirt.io/kubevirtci/cluster-up/cluster/kind-k8s-sriov-1.17.0/certcreator
```

Comment:
Applying this PR locally on kubevirt cause the following error on those two cases:
1.
``` 
$ cd cluster-up/cluster/kind-k8s-sriov-1.17.0/certcreator/
$ go build
```
2. `make cluster-up`

It fails on the following error:
```
certsecret.go:20:2: cannot find package "kubevirt.io/kubevirtci/cluster-up/cluster/kind-k8s-sriov-1.17.0/certcreator/certlib" in any of:
	/root/go/src/kubevirt.io/kubevirt/vendor/kubevirt.io/kubevirtci/cluster-up/cluster/kind-k8s-sriov-1.17.0/certcreator/certlib (vendor tree)
	/usr/local/go/src/kubevirt.io/kubevirtci/cluster-up/cluster/kind-k8s-sriov-1.17.0/certcreator/certlib (from $GOROOT)
	/root/go/src/kubevirt.io/kubevirtci/cluster-up/cluster/kind-k8s-sriov-1.17.0/certcreator/certlib (from $GOPATH)
```
Doing the same 2 commands at Kubevirtci folder works (both `go build` and `make cluster-up`)

It's because of this `go.mod` line i guess?

Reply:
I didnt manged to reproduce it, also it works on CI.
Maybe it something that related to GOPATH or GOROOT on your system

Reply:
It works for you because you have kubevirtci under `~/go/src/kubevirt.io/kubevirt/kubevirtci` imo,
and it shouldn't look there but it should use the `certlib` which is vendored under `cluster-up/cluster/kind-k8s-sriov-1.17.0/certcreator/certlib`
Please try to move `~/go/src/kubevirt.io/kubevirt/kubevirtci` and see if it happens.

Reply:
This is how I tried to reproduced and it works, I understand why you think it is related to the path but it dosent seem to affect I dont think we have `~/go/src/kubevirt.io/kubevirtci` on CI during pull-e2e-kubevirt jobs. 

It works because we specify this line (`module kubevirt.io/kubevirtci/cluster-up/cluster/kind-k8s-sriov-1.17.0/certcreator`) in go.mod
Then in the same project we use the module we created by adding `kubevirt.io/kubevirtci/cluster-up/cluster/kind-k8s-sriov-1.17.0/certcreator/certlib` to import on the go files you want to use it.
Theoretically we could change the name of the module in go.mod to whatever we want `jungle.io/banana/certcreator`
and use certlib by adding this import `jungle.io/banana/certcreator/certlib` in the file we want.

I think we shouldn't worry about regression, we can come up with better name later on

Reply:
Can you try please `find . -name selfsign.go` all over if you have it on other place?
For me locally it doesn't work, and it used to work, so I find it as a regression.
I would suggest and be glad please if it can be fixed in a way that will be agnostic to anything else beside the vendored folder (certlib), as its the correct way imho.